src/metrics/loc.rs

```diff
@@ -500,9 +500,59 @@ impl Loc for CppCode {
     }
 }
 
+impl fmt::Display for Java {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{:?}", self)
```

Comment:
I don't see similar code for other languages, is it a debug leftover ?
If it's useful in general, I guess it can be added for all languages in a follow-up PR.

Reply:
It indeed is from debugging. Ok to leave in then? Should I create an issue to track adding it to the others? Guss can discuss there and if we decide no, then I will remove this.

Reply:
I would prefer having that in an other PR for all the different languages.

Reply:
Will do. I will pull it out of this PR once the other items in this are sorted out. I have not found a better way of seeing what is going on.